foursight_core/react/api/aws_stacks.py

```diff
@@ -0,0 +1,133 @@
+import boto3
+import json
+from .datetime_utils import convert_utc_datetime_to_useastern_datetime_string
+from .misc_utils import memoize
+from collections import OrderedDict
+from dcicutils.obfuscation_utils import obfuscate_dict
+from typing import Optional, Union
+
+
+_STACK_NAME_PREFIX = "c4-"
+
+@memoize
+def aws_get_stacks() -> list:
+    """
+    Returns the list of all known AWS CloudFormation stacks with various metadata.
+    """
+    stacks_info = []
+    c4 = boto3.resource("cloudformation")
+    for stack in sorted(c4.stacks.all(), key=lambda key: key.name):
```

Comment:
See my comment on `aws_get_stack_resources` (elsewhere in this same file) about why `key` is the wrong parameter name and how to fix it.

Reply:
Got it, thanks.